### PR TITLE
No 'diff' option when binaries are involved (confirm-dialog).

### PR DESCRIPTION
**Pull request by fpauser**

### CODE REVIEW

**Comment by stefanpenner** [comment]
mind adding a unit test?

File: lib/models/file-info.js [hunk 1]
```diff
@@ -33,18 +33,30 @@ function diffHighlight(line) {
 }
 
 FileInfo.prototype.confirmOverwrite = function(path) {
+  var info = this;
```
Comment (Turbo87) [inline lib/models/file-info.js]: what's the reason for that extra variable?

Comment (fpauser) [inline lib/models/file-info.js]: You'r right - no reason/need to save the context there, will update the PR later.

**Comment by nathanhammond** [comment]
Can you do some git-fu to make sure that just your change appears in this PR? Not sure what else changed that you have non-passing tests, so hoping to reduce the surface area of this test.

**Comment by fpauser** [comment]
Did some gitfu to get it nice and dandy ;)
Locally I have one failing test that also breaks on master - did not investigate this further.

```
923 passing (21s)
  5 pending
  1 failing

1) Acceptance: ember generate passes custom cli arguments to blueprint options:
     AssertionError: expected "app/foo.js" to exist
```

**Comment by Turbo87** [comment]
@fpauser could you rebase this branch and see if the tests are still failing? Otherwise I'd like to get this merged soon.

**Comment by nathanhammond** [comment]
@homu r+

**Comment by nathanhammond** [comment]
@fpauser Thank you for making the user experience better!

**Pull request merged**